torch/_inductor/triton_heuristics.py

```diff
@@ -1114,6 +1114,16 @@ def persistent_reduction(size_hints, reduction_hint=False, meta=None, filename=N
                 size_hints, 2 * (256 // rnumel) if rnumel <= 256 else 1, rnumel
             )
         ]
+    # TODO: Check if this can be helpful in general.
+    # https://github.com/pytorch/pytorch/pull/111656
+    # This brings a speed-up with interpolate bilinear aa=True op on batched input
+    elif reduction_hint == ReductionHint.DEFAULT:
+        configs += [
+            triton_config_reduction(size_hints, xblock, rnumel)
+            for xblock in (256, 512)
+            if rnumel * xblock <= 4096 * 4 and xblock <= xnumel
+        ]
+
```

Comment:
This has to go in a separate PR and benchmarked on its own.
It'd be good to benchmark just adding 512 first, and then perhaps adding both, as this may increase compilation times of the models.

Reply:
Separate PR with adding 256 and 512 only is here: https://github.com/pytorch/pytorch/pull/111656